### ISSUE TITLE
webirc

### ISSUE THREAD

**Issue by JuggaloIRC** [KiwiChat/NextKiwiChat#2]
where do you enter the webirc pass for KiwiIRC ?

**Comment by JuggaloIRC** [comment]
I decided to switch to websocket support is there anyway to add direct websocket to the php file for wp users to connect?

**Comment by KiwiChat** [comment]
@JuggaloIRC  > where do you enter the webirc pass for KiwiIRC ?

Install [kiwiirc](https://github.com/kiwiirc/kiwiirc) together with [webirc gateway](https://github.com/kiwiirc/webircgateway) on your own server
example [WebIRC](https://www.unrealircd.org/docs/WebIRC_block) block  UnrealIRCD

**Comment by JuggaloIRC** [comment]
I have it on HatchetShack.com/JIRC with websocket but was wanting to integrate it with WordPress

**Comment by KiwiChat** [comment]
@JuggaloIRC

in this case, try the old generator, I abandoned it a long time ago [custom-kiwichat](https://custom.kiwichat.eu.org/)

![juggaloirc](https://github.com/KiwiChat/NextKiwiChat/assets/52504536/5917132a-e41a-463a-962e-cd64840713ab)

The generator takes a long time I hope you are patient

If in the future I am going to obtain or they are going to give me hosting, I will update the page

**Comment by JuggaloIRC** [comment]
It works I just wish I could get it to use logged in usernames, I got it on hatchetshack.com/juggaloirc

**Comment by KiwiChat** [comment]
@JuggaloIRC
you mean something like that [jirc](https://github.com/KiwiChat/jirc/archive/refs/tags/v1.0.zip) ?

**Comment by JuggaloIRC** [comment]
Yes that looks perfect I'll download it and set it up when I get home

**Comment by JuggaloIRC** [comment]
I just did an update for unrealircd and it broke websockets

**Comment by JuggaloIRC** [comment]
Websocket is working but chrome doesn't like the update

**Comment by JuggaloIRC** [comment]
I got it fixed, I had to update site/cookies to allow on a few options within the browsers
